tests/integration/test_model_download.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from requests import HTTPError
+
+from kagglehub import model_download
+
+HANDLE = "keras/bert/keras/bert_base_en/2"
+
+
+def test_successful_download():
```

Comment:
Tests should be in a class extension `unittest.TestCase`.

Each method should take a `self` argument and use assertions `self.assertEqual`.